_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+* Jordan Perr-Sauer, National Renewable Energy Laboratory. 
+
+The Steering Committee is currently meeting every two weeks and working on many of the activities below. In addition, we are planning a formal governance process that will include regular steering committee elections. The initial Steering Committee’s goal is to establish the framework for a long-term vibrant and robust community while defining the overall policy and direction of the US-RSE Association, based on the inputs of the US-RSE Association membership. The Steering Committee encourages and hopes that many new US-RSE Association activities and events will be initiated, organized, and managed by community committees and members at large.
+
+**RSE defined**. We [defined](https://us-rse.org/what-is-an-rse/) Research Software Engineers inclusively, encompassing "all those who regularly use expertise in programming to advance research. This includes researchers who spend a significant amount of time programming, full-time software engineers writing code to solve research problems, and those somewhere in-between. We aspire to apply the skills and practices of software development to research to create more robust, manageable, and sustainable research software."
+
+**Mission**. We've defined an initial mission: The US-RSE Association is centered around three main goals. We aim to target our activities and actions to serving these three aspects:
+
+**Community**: We seek to provide a coherent association of those who identify with the role (not necessarily title) of Research Software Engineer based on our inclusive definition. This group aims to provide the members of the community the ability to share knowledge, professional connections, and resources.
+
+**Advocacy**: We aim to promote RSEs’ impact on research, highlighting the increasingly critical and valuable role RSEs serve.
+
+**Resources**: We strive to provide useful resources to multiple demographics. For current and future RSEs we strive to provide technical and career development resources to support their professional development. We aim to provide access to information and material to support the establishment and expansion of RSE positions and groups within the research ecosystem including the material for making business justifications for RSEs.
+
+**Logo, website, Slack**. We've created a [US-RSE logo](https://github.com/USRSE/logo) and a [US-RSE website](https://us-rse.org/) that's hosted on [GitHub](https://github.com/USRSE/usrse.github.io) and has an [open method for people to contribute](https://github.com/USRSE/usrse.github.io/blob/master/README.md) jobs, events, and other content. As part of the website, we build a map of RSE member locations. We've also created an RSE Slack team with channels for different topics of interest that enable US-RSE members to chat with each other. As of 22 April 2020, the Slack team, which includes both members and others who are interested in US-RSE issues, has 522 users. 
```

Comment:
> build a map of RSE member locations -> build a map of US-RSE member locations
and
> an RSE Slack team -> a US-RSE Slack team

These are the spots I think we mean to say "US-RSE member locations."

Reply:
fixed